ingester/src/wal/wal_sink.rs

```diff
@@ -219,7 +219,7 @@ mod tests {
 
         // Obtain all the ops in the file
         let mut ops = Vec::new();
-        while let Ok(Some(mut batch)) = reader.next_batch() {
+        while let Some(Ok(mut batch)) = reader.next() {
```

Comment:
If `reader.next()` returns an error, I think it may be ignored here (though that was not introduced by this PR)

Perhaps you could use something like this to check for the error (not tested):

```suggestion
        while let Some(mut batch) = reader.next().transpose()? {
```

Or perhaps (as this is a test):

```suggestion
        while let Some(mut batch) = reader.next().unwrap() {
```